WalletWasabi.Fluent/ViewModels/ApplicationViewModel.cs

```diff
@@ -57,8 +57,7 @@ public ApplicationViewModel(IMainWindowService mainWindowService)
 
 	public void OnShutdownPrevented()
 	{
-		RxApp.MainThreadScheduler.Schedule(async () =>
-			await MainViewModel.Instance.CompactDialogScreen.NavigateDialogAsync(new ShuttingDownViewModel(this)));
+		RxApp.MainThreadScheduler.Schedule(() => MainViewModel.Instance.CompactDialogScreen.To(new ShuttingDownViewModel(this)));
```

Comment:
Is the Scheduling needed?

Reply:
@soosr From looking at the code that invokes `OnShutdownPrevented()` it seems that it always runs on the UI thread, so at first glance I would think no, but it doesn't create any problems either, and I think it's more future proof to keep it, because if at any time in the future there is an invocation of that method outside the UI thread you will get a crash otherwise.

Let me know your thoughts on this.